pkg/checkout/session.go

```diff
@@ -0,0 +1,86 @@
+package checkout
+
+import (
+	"bufio"
+	"encoding/json"
+	"io"
+	"os"
+
+	"github.com/stripe/stripe-cli/pkg/config"
+	"github.com/stripe/stripe-cli/pkg/requests"
+)
+
+type response struct {
+	ID string `json:"id"`
+}
+
+func createCheckoutSession(cfg *config.Config) (string, error) {
+	secretKey, err := cfg.Profile.GetAPIKey(false)
+	if err != nil {
+		return "", err
+	}
+
+	data := requests.RequestParameters{}
+	data.AppendData([]string{
+		`success_url=http://localhost:4242/success`,
+		`cancel_url=http://localhost:4242/cancel`,
```

Comment:
Shouldn't these two URLs be parameterized with the port number?

Reply:
Oh, good catch! Fixing